src/etc/inc/openvpn.inc

```diff
@@ -719,6 +719,15 @@ function openvpn_reconfigure($mode, $settings) {
 				break;
 		}
 	}
+	if ($settings['dev_mode'] === 'tun' && isset($config['unbound']['enable']) && isset($config['unbound']['regovpnclients'])) {
```

Comment:
OK, that makes sense. Though you should then also reflect this limitation in the help/description of the GUI option.